components/renku_data_services/storage/rclone.py

```diff
@@ -176,36 +176,10 @@ def __add_webdav_based_storage(
     ) -> None:
         """Create a modified copy of WebDAV storage and add it to the schema."""
         # Find WebDAV storage schema and create a modified copy
-        storage_copy = RCloneValidator.__find_webdav_storage(spec)
+        storage_copy = RCloneValidator.__find_storage(spec, "webDav")
```

Comment:
```suggestion
        storage_copy = RCloneValidator.__find_storage(spec, "webdav")
```
I think this is a typo? at least it was lower case before.

Reply:
yes, it is. thanks!